### PR TITLE
Bump version to 5.0.1

### PR DESCRIPTION
**Pull request by jasontedor**
This commit bumps the version number to 5.0.1.

### CODE REVIEW

File: dev-tools/create_bwc_index.py [hunk 1]
```diff
@@ -265,12 +265,21 @@ def generate_index(client, version, index_name):
...
         'field.with.dots': {
           'type': 'string',
           'boost': 4
         }
       })
+    else:
+      mappings["doc"]['properties'].update({
+        'field.with.dots': {
+          'type': 'text',
+          'index': True
```
Comment (bleskes) [inline dev-tools/create_bwc_index.py]: is this not the default?

**Review by bleskes — APPROVED** [review]
LGTM. left a minor question

File: dev-tools/create_bwc_index.py (hunk 1 above)
Comment (jasontedor) [inline dev-tools/create_bwc_index.py]: @bleskes You mean on a dynamically created field? I think the default is to create both a `text` and a `keyword` field. Or are you asking about `'index': True`?

**Comment by s1monw** [comment]
LGTM left one question

File: dev-tools/create_bwc_index.py (hunk 1 above)
Comment (jasontedor) [inline dev-tools/create_bwc_index.py]: I pushed b009243d660b9f3fb554c42920f655ca0f09da63.

**Comment by jasontedor** [comment]
Thanks for reviewing @bleskes and @s1monw. I had to push 54ded914a6bea5037999e516dc76230620571fb2 to account for changes from 2.x to 5.0.0. Can you please review?

**Comment by s1monw** [comment]
LGTM thanks for the fixes

**Comment by bleskes** [comment]
LGTM2

**Comment by bleskes** [comment]
asking about the index=true part

**Pull request merged**